addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -6555,274 +6555,139 @@ stock GetBossIndex(client)
 
 stock ParseFormula(client, const String:key[], const String:defaultFormula[], defaultValue)
 {
-	decl String:formula[1024], String:buffer[2];
-	new String:strValue[1024], Float:sum[128], calcOperators:_operator[128];
-	new bool:closeBracket, bool:usePlayers, bool:foundOperator, bracket;
+	decl String:formula[1024], String:bossName[1024];
```

Comment:
Yeah there's no way the boss name is ever going to be 1024 characters.  Could you use the "regular" size for boss names (Ctrl+F'ing bossName should be enough)